bin/varnishncsa/varnishncsa.c

```diff
@@ -895,12 +895,19 @@ parse_format(const char *format)
 
 static int
 isprefix(const char *prefix, size_t len, const char *b,
-    const char *e, const char **next)
+    const char *e, const char **next, const char *sep)
 {
 	assert(len > 0);
 	if (e < b + len || strncasecmp(b, prefix, len))
 		return (0);
 	b += len;
+	if (sep != NULL) {
```

Comment:
use `strpbrk()`?

Reply:
You have 3 legitimate separators with the new prefix definition in varnishncsa: `':'`, `' '` and `'\t'` (VSL prefix and field separators).

Reply:
Sorry @dridi, I agree that we should be consistent. But are we really using `/t` anywhere in VSL?